client/Containers/Cart/Cart.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import ProductCard from './ProductCard';
+import Summary from './Summary';
+import styled from 'styled-components';
+
+const mapStateToProps = (state) => {
+  return {
+    cart: state.cart,
+  };
+};
+
+class Cart extends Component {
+  itemCount = () => {
+    let {
+      cart: { cartProducts },
+    } = this.props;
+    cartProducts = cartProducts || [];
+    if (!cartProducts.length) return 0;
+    return cartProducts.reduce((accum, item) => accum + item.quantity, 0);
+  };
+
+  render() {
+    const { cart } = this.props;
+    const { itemCount } = this;
+    let { cartProducts } = cart;
+    cartProducts = cartProducts || [];
+    return (
+      <>
+        <Row id="cart">
+          <Child>
+            <h1>Cart: ({`${itemCount()}`})</h1>
+            <hr className="heavy"></hr>
+            {cartProducts.length ? (
+              cartProducts.map((cartProduct) => (
+                <ProductCard
+                  cartProduct={cartProduct}
+                  key={cartProduct.productId}
+                />
```

Comment:
Passing in the cartProduct object directly to the ProductCard, rather than giving it an id that it uses to find the product